### DISCUSSION TITLE
video script behavior question

### DISCUSSION THREAD

**Discussion by d3baser** [RandomNinjaAtk/docker-lidarr-extended#227] [General]
First off, I really appreciate the effort. These scripts add a lot to lidarr functionallity and I'm considering switching all of my *arr dockers over. 

My q is if it is possible to configure the video script to 'upgrade' prior downloads. This is an edge case, but after using the extension for a while I have about 550gb of videos downloaded from youtube, but recently got tidal. The video wiki says "tidal is preferred, if you both a youtube link and tidal link is found)" Is there any way to automatically replace youtube grabs with tidal versions when available, or is my best option to nuke the video directory and history?

**Comment by RandomNinjaAtk** [comment]
Just going by memory but I believe it already does that... However, you'll find that not many tidal videos are linked on musicbrainz, so your mileage will vary on how many you get because the quality of the indexer... Unless your willing to take the time to improve it.

Edit:
Also see #208... 

Search is your friend...

**Reply by d3baser** [reply]
Thanks for the reply. I searched but came up empty.  Now I see it's bc I only searched discussions, not issues. 

I'll start linking tidal videos on MB.
